tests/test_simple.py

```diff
@@ -95,15 +95,6 @@ def test_no_pwd(self):
         )
         self.assertFalse(result)
 
-    @defer.inlineCallbacks
-    def test_uppercase_username(self):
```

Comment:
I removed this test as I added it in the last PR and is no longer relevant to how the code operates.